### ISSUE TITLE
Modded HUD shows incorrect pager amount in stealth

### ISSUE THREAD

**Issue by vadcx** [steam-test1/VPlusHUD#284]
This happened to me on the latest version, but the description is poor.

I joined into an active mission, either the guard with a pager was being shot down, or answered. The VPlusHUD ended up showing 3 pagers even though both players told me we're out of pagers.

Indeed: the TAB menu showed 4/4 pagers used, while the HUD showed only 3.

Probably the current code should do the same as game's TAB does to retrieve the current pager amount.

**Comment by steam-test1** [comment]
Could you look into updating it, as I don't play multiplayer to test it

**Comment by vadcx** [comment]
Yes of course I write these as a personal-public issue tracker. I appreciate you considering this, but my intent is to not forget about them. And even if I don't find the time to fix it myself, I don't want a particular issue to go unnoticed and untracked.

I'm looking into this and already having trouble to locate vanilla's code for the left Tab menu :)

**Comment by steam-test1** [comment]
You can use the code in TabStat.lua just search pager

**Comment by vadcx** [comment]
Thanks I've found it in game code, but there might be a deeper issue. While VHUDPlus doesn't correctly determine the pager count... The issue is the mod's detection is event-based and if it doesn't count correctly, it means an event is being missed. This is never OK and I don't want to hide this issue by only fixing the pager counter.

Vanilla pager answering interaction: `copbrain.lua` - `function CopBrain:on_alarm_pager_interaction(status, player)`
Our event-based registration: https://github.com/steam-test1/VPlusHUD/blob/b561f0a5441f8cc1bb1c5c7d8a00e10501e10aa2/devlua/HUDList.lua#L3095

get_pagers(key) is simply this:

```lua
	function GameInfoManager:get_pagers(key)
		if key then
			return self._pagers[key]
		else
			return self._pagers
		end
	end
```

I will try to reproduce this and see if I can understand the root cause and where the event goes missing.

For now I only pushed the code that would count the pagers like the game: https://github.com/vadcx/VPlusHUD/tree/pr/wrong-pager-counter

**Comment by steam-test1** [comment]
Could you test the following code
```lua
self:set_count(table.size(managers.gameinfo:get_pagers()))
```
to
```lua
self:set_count(managers.groupai:state():get_nr_successful_alarm_pager_bluffs())
```

**Comment by vadcx** [comment]
Yes essentially the code you suggested is in the [pr/wrong-pager-counter](https://github.com/vadcx/VPlusHUD/tree/pr/wrong-pager-counter) link from above. I will test it with a friend (or myself now, really, since I can get a second copyof the game for free)

To confirm: yes, this appears to happen all the time if you join a game in progress, i.e. for 1/4 pagers used (according to players and tab), VPlusHUD showed no HUD element at all.
